### PR TITLE
Add option to change language of the site

### PR DESCRIPTION
**Pull request by m4v15**
relates to issue #134 #100 #20

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,23 +2,32 @@ const path = require('path')
 const express = require('express')
 const hbs = require('express-handlebars')
 const router = require('./routes/index.js')
-const text = require('./text.js').english
+var bodyParser = require('body-parser')
```
Comment (hibaakel) [inline app.js]: why using var instead of const

File: app.js [hunk 2]
```diff
@@ -2,23 +2,32 @@ const path = require('path')
...
+const dir = 'rtl'
 
-app.set('port', process.env.PORT || 4444)
+app.locals.dir = dir
+app.locals.text = text
+app.locals.lang = lang
 
 const pubPath = path.join(__dirname, './', 'public')
 
+app.use(bodyParser.urlencoded({extended: false}))
```
Comment (hibaakel) [inline app.js]: what this line do ?

Comment (m4v15) [inline app.js]: it means post requests are parsed automatically or something

File: app.js (hunk 1 above)
Comment (m4v15) [inline app.js]: done!

File: app.js (hunk 2 above)
Comment (esraajb) [inline app.js]: https://github.com/expressjs/body-parser

**Pull request merged**